common/vpcclient/models/share.go

```diff
@@ -30,6 +30,7 @@ type Share struct {
 	Size          int64          `json:"size,omitempty"`
 	Iops          int64          `json:"iops,omitempty"`
 	EncryptionKey *EncryptionKey `json:"encryption_key,omitempty"`
+	Bandwidth     *int64         `json:"bandwidth,omitempty"`
```

Comment:
Why we need pointer here ? Is it just to ignore the field if it is nil , instead of passing zero in case of non-pointer case.